.travis.yml

```diff
@@ -45,7 +45,7 @@ before_script:
 script:
   - cd ../build
   - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then export LDFLAGS=" -L/usr/local/opt/qt5/lib ${LDFLAGS}"; export CPPFLAGS=" -I/usr/local/opt/qt5/include ${CPPFLAGS}"; fi
-  - if [ "${Q_OR_C_MAKE}" = "qmake" ]; then qmake -v; qmake ../src/src.pro && make -j2; else cmake --version; cmake .. && make -j2 && make check; fi
+  - if [ "${Q_OR_C_MAKE}" = "qmake" ]; then qmake -v; qmake ../mudlet.pro && make -j2; else cmake --version; cmake .. && make -j2 && make check; fi
```

Comment:
I'm not into this idea because it would mean everyone compiling Mudlet would stop being able to do so and be really confused until they figure out that we introduced a new, high-level makefile. Is this absolutely mandatory and can we make edbee work without causing a lot of pain to everybody?

Yes, we'll update the instructions, but everyone knows that nobody reads the instructions.